examples/http_redirect.py

```diff
@@ -36,9 +35,7 @@ def proxy(request, path):
 
 @https.main_process_start
 async def start(app, _):
-    http_server = await http.create_server(
-        port=HTTP_PORT, return_asyncio_server=True
-    )
+    http_server = await http.create_server(port=HTTP_PORT, return_asyncio_server=True)
```

Comment:
Line length updated from Sanic prior setting of 79 to Ruff's default 88, which caused some reformatting like this. Additionally Ruff doesn't count pragma comments to line length, so even without the change it can make some lines longer.